distributions/aws/aws-alb-ingress-controller/base/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+# Application Load Balancer (ALB) Ingress Controller Deployment Manifest.
+# This manifest details sensible defaults for deploying an ALB Ingress Controller.
+# GitHub: https://github.com/kubernetes-sigs/aws-alb-ingress-controller
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: alb-ingress-controller
+  # Namespace the ALB Ingress Controller should run in. Does not impact which
+  # namespaces it's able to resolve ingress resource for. For limiting ingress
+  # namespace scope, see --watch-namespace.
+#  namespace: kubeflow
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: alb-ingress-controller
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: alb-ingress-controller
+      annotations:
+        sidecar.istio.io/inject: "false"
+    spec:
+      containers:
+        - name: alb-ingress-controller
+          args:
+            # Limit the namespace where this ALB Ingress Controller deployment will
+            # resolve ingress resources. If left commented, all namespaces are used.
+            # - --watch-namespace=your-k8s-namespace
+
+            # Setting the ingress-class flag below ensures that only ingress resources with the
+            # annotation kubernetes.io/ingress.class: "alb" are respected by the controller. You may
+            # choose any class you'd like for this controller to respect.
+            - --ingress-class=alb
+
+            # REQUIRED
+            # Name of your cluster. Used when naming resources created
+            # by the ALB Ingress Controller, providing distinction between
+            # clusters.
+#            - --cluster-name=$(CLUSTER_NAME)
+            - --cluster-name=$(CLUSTER_NAME)
+
+            # AWS VPC ID this ingress controller will use to create AWS resources.
+            # If unspecified, it will be discovered from ec2metadata.
+            # - --aws-vpc-id=vpc-xxxxxx
+
+            # AWS region this ingress controller will operate in.
+            # If unspecified, it will be discovered from ec2metadata.
+            # List of regions: http://docs.aws.amazon.com/general/latest/gr/rande.html#vpc_region
+            # - --aws-region=us-west-1
+          # Repository location of the ALB Ingress Controller.
+          image: docker.io/amazon/aws-alb-ingress-controller
```

Comment:
should we fix the tag of the image to avoid any break in future?

Reply:
It is fixed here: https://github.com/awslabs/kubeflow-manifests/pull/25/files#diff-13d4cf72d8bd3b50fc39291dde98e8569a1c9ccc616a725cdc99a4a97108905fR15-R17

Reply:
Why there why not here?

Reply:
is it because `kustomization` file is one layer above this ?

Reply:
maybe the other files are replica of official installer and kustomization file and deployment_patch contains the overrides making it easy to identify changes

I say maybe because I haven't looked into the official 1.1.5 installer. We should look into this while upgrading it to v2+